credit/metrics.py

```diff
@@ -100,3 +102,203 @@ def __call__(self, pred, y, clim=None, transform=None, forecast_datetime=0):
+
+                # Compute anamolies
+                for i, var in enumerate(ordered_acc_vars):
+                    clim = self.get_climatology(forecast_datetime, var).to(
+                        dtype=pred.dtype, device=pred.device
+                    ).unsqueeze(0)
+                    anomalies_pred.append(acc_pred[:, i] - clim)
+                    anomalies_y.append(acc_y[:, i] - clim)
+
+                anomalies_pred = torch.stack(anomalies_pred, dim=1)
+                anomalies_y = torch.stack(anomalies_y, dim=1)
+
+                for i, var in enumerate(self.acc_vars):
+                    pred_prime = anomalies_pred[:, i] - torch.mean(anomalies_pred[:, i])
+                    y_prime = anomalies_y[:, i] - torch.mean(anomalies_y[:, i])
+
+                    # Add epsilon to avoid division by zero
+                    epsilon = 1e-7
+                    denominator = (
+                        torch.sqrt(
+                            torch.sum(w_var * w_lat * pred_prime**2)
+                            * torch.sum(w_var * w_lat * y_prime**2)
+                        )
+                        + epsilon
+                    )
+                    loss_dict[f"acc_{var}"] = (
+                        torch.sum(w_var * w_lat * pred_prime * y_prime) / denominator
+                    )
+                anamoly_scores = True
+
+            # Compute RMSE, MSE, MAE for all vars
+            error = pred - y
+            for i, var in enumerate(self.vars):
+                loss_dict[f"rmse_{var}"] = torch.mean(
+                    torch.sqrt(
+                        torch.mean(error[:, i] ** 2 * w_lat * w_var, dim=(-2, -1))
+                    )
+                )
+                loss_dict[f"mse_{var}"] = (error[:, i] ** 2 * w_lat * w_var).mean()
+                loss_dict[f"mae_{var}"] = (
+                    torch.abs(error[:, i]) * w_lat * w_var
+                ).mean()
+
+            # Compute average metrics
+            if anamoly_scores:
+                loss_dict["acc"] = np.mean(
+                        [loss_dict[k].cpu().item() for k in loss_dict.keys() if "acc_" in k]
+                )
+            loss_dict["rmse"] = np.mean(
+                [loss_dict[k].cpu().item() for k in loss_dict.keys() if "rmse_" in k]
+            )
+            loss_dict["mse"] = np.mean(
+                [loss_dict[k].cpu().item() for k in loss_dict.keys() if "mse_" in k]
+            )
+            loss_dict["mae"] = np.mean(
+                [loss_dict[k].cpu().item() for k in loss_dict.keys() if "mae_" in k]
+            )
+
+        return loss_dict
+
+
+if __name__ == "__main__":
+    import yaml
+    import logging
+    import xarray as xr
+    from credit.parser import credit_main_parser
+
+    logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    )
+    logger = logging.getLogger(__name__)
+
+    # Open an example config
+    with open(
```

Comment:
Please point example config to relative path in repo.

Reply:
It should now point to example-v2025.2.0.yml in /config